docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -141,30 +147,30 @@ Now that your plugin is ready to run, follow the instructions bellow for your ID
 
 1. Run the config (with or without the debugger).
 
-1. Start Grafana, if it's not already running.
+1. If Grafana isn't already running, run it.
 
-> If you re-run the configuration, Grafana will automatically reload the plugin.
+> **Note:** If you re-run the configuration, Grafana  automatically reloads the plugin.
 
 ### Other IDEs
 
 Configure your code editor to run the following steps:
 
-1. Build the executable with debug flags.
+1. Build the executable file with debug flags.
    ```
    mage build:debug
    ```
-1. Run the plugin's executable (inside `dist`) with `-standalone -debug` flags.
+1. Run the plugin's executable file (inside `dist`) with `-standalone -debug` flags.
    ```
    ./gpx_xyz_linux_amd64 -standalone -debug
    ```
 1. Attach a debugger to the process.
 
-Then, you can start Grafana, if it's not already running.
+1. If Grafana isn't already running, run it.
 
-> If you re-run the configuration, Grafana will automatically reload the plugin.
+> If you re-run the configuration, Grafana automatically reloads the plugin.
 
 ### Notes
 
-- All logs are printed in the plugin's `stdout` rather than in Grafana logs.
-- If the backend plugin doesn't serve requests after turning off debug mode, you can force reset the standalone mode. To do so, delete the files `dist/standalone.txt` and `dist/pid.txt` alongside the executable file, and then restart Grafana.
-- We currently do not support debugging backend plugins running inside Docker.
+- Grafana prints all logs in the plugin's `stdout` rather than in Grafana logs.
```

Comment:
@xnyo What do you think?

Reply:
@josmperez the logging is done by the plugin because Grafana is running in a separate process.

I added this note to let the developer know they should look for logs into the plugin's console rather than Grafana's console

Reply:
Reverted my change.